hazelcast/src/test/java/com/hazelcast/config/NearCacheConfigAccessorTest.java

```diff
@@ -36,27 +36,16 @@ public void testConstructor() {
     }
 
     @Test
-    public void testInitDefaultMaxSizeForOnHeapMaps_whenNull_thenDoNothing() {
-        NearCacheConfigAccessor.initDefaultMaxSizeForOnHeapMaps(null);
+    public void testCopyInitDefaultMaxSizeForOnHeapMaps_whenNull_thenDoNothing() {
```

Comment:
Can you also add a test that, when the size is configured, we are not copying the near cache config?